src/DataXml/Utd820/ON_NSCHFDOPPR_UserContract_820_05_01_01.cs

```diff
@@ -1584,39 +1596,39 @@ public enum TaxRateWithTwentyPercentAndTaxedByAgent {
         
         /// <remarks/>
         [System.Xml.Serialization.XmlEnumAttribute("без НДС")]
-        безНДС,
+        WithoutVat,
         
         /// <remarks/>
         [System.Xml.Serialization.XmlEnumAttribute("0%")]
-        Item0,
+        Zero,
         
         /// <remarks/>
         [System.Xml.Serialization.XmlEnumAttribute("10%")]
-        Item10,
+        TenPercent,
         
         /// <remarks/>
         [System.Xml.Serialization.XmlEnumAttribute("18%")]
-        Item18,
+        EighteenPercent,
         
         /// <remarks/>
         [System.Xml.Serialization.XmlEnumAttribute("20%")]
-        Item20,
+        TwentyPercent,
```

Comment:
Percent18, Percent20 can be more familiar (like in protobuf). At your discretion

Reply:
I think it's not so neccesary